### DISCUSSION TITLE
How to give image tar file as input to trivy binary

### DISCUSSION THREAD

**Discussion by chakradharv** [aquasecurity/trivy#2067] [Q&A] [answered]
In the initial release version of v0.18.3 to list of packages and vulnerabilities of the tar file I used to run this command 
`./trivy_binary --input tar file`. This is used to list packages and vulnerabilities. But with the latest release of trivy when I run this command I am getting this error `Incorrect Usage. flag provided but not defined: -input`. Could someone please give info about how to list of packages and vulnerabilities of a given tar file with the trivy.  This would be of great help to me

**Comment by NitroCao** [comment, marked as answer]
If you run `trivy image --help` command, you'll find there is an option called `--input` which is used to specify the input file path instead of image name. You can also set the value of `--input` option via `TRIVY_INPUT` environment variable.

**Reply by chakradharv** [reply]
@NitroCao Thanks a lot.
